cg/services/order_validation_service/errors/case_sample_errors.py

```diff
@@ -143,3 +143,8 @@ class CaptureKitMissingError(CaseSampleError):
 class WellFormatError(CaseSampleError):
     field: str = "well_position"
     message: str = "Well position must follow the format A-H:1-12"
+
+
+class ContainerNameRepeatedError(CaseSampleError):
+    field: str = "container_name"
+    message: str = "Container name must be unique among samples"
```

Comment:
Thought that hit me now and which should be patched for the samples PR as well - we should have the message highlighting that tube names needs to be unique. Container names are allowed to be repeated, especially for well plate samples.